core/src/OpenMPTarget/Kokkos_OpenMPTarget_Instance.cpp

```diff
@@ -81,13 +80,6 @@ int OpenMPTargetInternal::concurrency() const {
 #elif defined(KOKKOS_ARCH_PASCAL60) || defined(KOKKOS_ARCH_PASCAL61)
   max_threads = max_threads_sm * 60;
 #endif
-#elif defined(KOKKOS_ARCH_INTEL_GPU)
```

Comment:
What does this have to do with compiler support?
Does LLVM Clang not support offloading on Intel GPUs?

Reply:
Yes. LLVM clang only supports openmp offloading on NVIDIA and AMD GPUs.

Only Intel compilers support offloading on Intel GPUs.